judgels-frontends/raphael/src/routes/uriel/contests/single/problems/single/ContestProblemPage/ContestProblemPage.tsx

```diff
@@ -44,21 +58,46 @@ interface ContestProblemPageState {
   defaultLanguage?: string;
   languages?: string[];
   problem?: ContestProblem;
+  problemInfo?: ProblemInfo;
   totalSubmissions?: number;
-  worksheet?: ProblemWorksheet;
+  bundleLatestSubmission?: { [id: string]: ItemSubmission };
+  worksheet?: ProgrammingProblemWorksheet | BundleProblemWorksheet;
+  problemType?: ProblemType;
 }
 
 export class ContestProblemPage extends React.Component<ContestProblemPageProps, ContestProblemPageState> {
   state: ContestProblemPageState = {};
 
   async componentDidMount() {
-    const { defaultLanguage, languages, problem, totalSubmissions, worksheet } = await this.props.onGetProblemWorksheet(
+    const problemInfo = await this.props.onGetProblemInfo(this.props.contest.jid, this.props.match.params.problemAlias);
```

Comment:
But there is no `ContestStyle.Bundle` yet. Should I just create it?

Reply:
Actually we need `ProblemInfo` to get bundle problem's title. Unlike the programming problem that only need worksheet. Because in bundle problem, the title information is in the `ProblemInfo` instead of worksheet.
So, what I did is removing title and alias from rendering function.

Reply:
Hmm. Can we make the `ProblemWorksheet` for bundle problem also return a `String problemTitle` / `ProblemStatement statement`? @nathanchrs

Reply:
Because @nathanchrs already add problem statement to the worksheet, now I can add tite to the problem statement card.